lib/models/project.js

```diff
@@ -338,7 +339,8 @@ Project.prototype.supportedInternalAddonPaths = function() {
     path.join(internalMiddlewarePath, 'tests-server'),
     path.join(internalMiddlewarePath, 'history-support'),
     path.join(internalMiddlewarePath, 'serve-files'),
-    path.join(internalMiddlewarePath, 'proxy-server')
+    path.join(internalMiddlewarePath, 'proxy-server'),
+    path.join(this.emberCliNodeModulesPath, 'ember-cli-legacy-blueprints')
```

Comment:
does that change mean that our users don't have to add the addon themselves and it will be added automatically without an explicit dependency on their side?